README.md

```diff
@@ -80,77 +80,82 @@ The platform is built on top of [Ambire Wallet](https://www.ambire.com) and uses
+* **No custody of funds:** Users have complete control over their funds, allowing them to withdraw any amount at any time without any minimum threshold requirements. 
+* **Ease of use:** Navigating modern ad tech can be complex, but AdEx simplifies the process with its user-friendly interface, ensuring effortless interaction with the platform. 
+* **Top-tier support.** We are committed to providing our advertisers with a smooth and hassle-free experience. Our dedicated team is available to assist with product inquiries, targeting strategy guidance, and campaign setup to ensure a seamless user experience.
+* **Expanding the Web3 user base** We enable advertisers to extend their reach to an untapped segment of potential crypto users, paving the way for new business prospects and opportunities for growth and expansion and reaching the next billion users.
 
 ### Terminology
 
 
+#### Platform 
+We use “platform” to refer to the AdEx DSP
+
 #### Supply side
 
-Throughout these documents, "supply side", "publisher" or "publishers" all refer to entities who sell ad inventory.
+Throughout these documents, "supply side", "SSP", "publisher" or "publishers" all refer to entities who sell ad inventory.
 
 #### Demand side
 
-Throughout these documents, "demand side", "advertiser", "advertisers" or "buyers" all refer to entities who buy ad inventory.
+Throughout these documents, "demand side", "DSP", "advertiser", "advertisers" or "buyers" all refer to entities who buy ad inventory.
 
-#### Users
+#### Programmatic 
 
-When we refer to "users", we mean end users: not of AdEx itself, but of the publishers. In other words, the users who see the ads, but might not even be aware of AdEx's existence.
+Programmatic advertising is the automated buying and selling of digital ad inventory, using real-time bidding and data-driven targeting, to deliver the right ad to the right audience at the right time.
 
-<div class='break-page'></div>
+#### Demand-side platform (DSP) 
+A DSP (Demand-side Platform) is a technology platform that allows advertisers to buy and manage digital ad inventory programmatically.
 
-#### Events
+#### Supply-side platform (SSP)
+An SSP (Supply-Side Platform) is a technology platform that enables publishers to sell and manage their digital ad inventory programmatically. 
 
-Events, in the context of the AdView or the off-chain event aggregation, mean anything that a user does in regard to a digital ad - for example, click, impression, closing of the web page, etc. Events are usually broadcast as signed messages.
+#### Direct publishers
+Direct publishers are websites that have a direct connection to AdEx’s DSP, bypassing the need for a DSP or ad exchange platform. 
 
-#### Custom events
+#### Users
 
-Custom events usually refer to events that are publisher-defined. For example, if you're a publisher with an e-commerce website, you might choose to send an event for product purchases.
```

Comment:
custom events can still be a thing. And they should be. Especially in directly integrated (premium) publishers